doc/doc-txt/ChangeLog

```diff
@@ -277,7 +277,7 @@ JH/21 Bug 1720: Add support for priority groups and weighted-random proxy
       "pri" and "weight".  Note that the previous implicit priority given by the
       list order is no longer honoured.
 
-JH/22 Bugs 963, 1721: Fix some corner cases in message body canonicalisation
+JH/22 Bugs 963, 1721: Fix some corner cases in message body canonicalization
```

Comment:
[en-US] I'm told that current coders tend to en-GB, in which case this and a number of other commits can be dropped.